.vscode/settings.json

```diff
@@ -1,4 +1,10 @@
 {
+    "editor.rulers": [88],
+
+    "files.exclude": {
+      "**/__pycache__": true,
```

Comment:
Why is this needed?

Reply:
@dlech the __pycache__/*.pyc file shows up in the VSC browser when the developer (e.g., I) uses the library in the dev/editable mode. I think it's a good idea to hide these generated files from the developer.